packages/multi-select-combo-box/src/vaadin-multi-select-combo-box-scroller.js

```diff
@@ -24,14 +24,11 @@ class MultiSelectComboBoxScroller extends ComboBoxScroller {
     this.setAttribute('aria-multiselectable', 'true');
   }
 
-  /** @private */
-  __getAriaSelected(_focusedIndex, itemIndex) {
-    const item = this.items[itemIndex];
-    return this.__isItemSelected(item, null, this.itemIdPath).toString();
-  }
-
-  /** @private */
-  __isItemSelected(item, _selectedItem, itemIdPath) {
+  /**
+   * @protected
+   * @override
+   */
+  _isItemSelected(item, _selectedItem, itemIdPath) {
```

Comment:
On multi-select-combo-box, for some reason, VO always reads `not-selected, selected` for not selected items and `selected, selected` for selected items. This can be handled separately from this PR:

![Screenshot 2023-01-26 at 17 39 30](https://user-images.githubusercontent.com/1222264/214880409-bfc97bda-bb33-4f9a-9605-e89fd309a65f.png)

Reply:
Yeah, it seems to be a not related issue since it also happens on `master`.